docs/getting-started/1242-install.mdx

```diff
@@ -156,6 +157,41 @@ If you want to dig into the details of how Dagger works, we recommend you read o
 
 <Button label="How It Works" docPath="f44rm/how-it-works" />
 
+export const Templates = ({ templates }) => {
+  return (
+    <div className={styles.templates}>
+      {templates.map((x, index) => (
+        <TemplateCard
+          key={index}
+          label={x.label}
+          imgFilename={x.imgFilename}
+          url={x.url}
+          dir={x.dir}
+          cmd={x.cmd}
+        />
+      ))}
+    </div>
+  );
+};
+
+export const TemplateCard = ({ label, imgFilename, url, dir, cmd }) => {
```

Comment:
neat: no need to create a component. We should just use html in situ

Reply:
Same as Template